tests/BunqSdkTestBase.php

```diff
@@ -75,6 +77,13 @@ class BunqSdkTestBase extends TestCase
      */
     protected $cashRegister;
 
+    /**
+     * Spending money constants.
+     */
+    const SPENDING_MONEY_AMOUNT = '500';
+    const SPENDING_MONEY_RECIPIENT = 'sugardaddy@bunq.com';
+    const SPENDING_MONEY_DESCRIPTION = 'sdk php test, thanks daddy <3 - OG';
```

Comment:
Why `- OG`? I'd suggest to keep it clean.

Reply:
Where's the fun in that? 😜